### PR TITLE
fix #72 exercise on code block in markdown

### LINKED ISSUE #72: Episode 02, exercise "Links" - code block in the exercise body is redundant
Not quite sure but feels like code block in the exercise body is redundant and should only be in the solution body.

Screenshot 2020-10-30 at 17 34 30

### PR DESCRIPTION
**Pull request by annefou**
Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  

---

### CODE REVIEW

File: _episodes/markdown.md [hunk 1]
```diff
@@ -187,34 +187,23 @@ Syntax highlighting of code blocks can increase readability a lot.
...
-> <a href=https://carpentries.org/>This is a link</a>
-> </body>
-> </html>
-> ```
-> ~~~
-> {: .source }
->
-> then guess in which programming language each snippet is written,
-> and fill the gaps to activate syntax highlighting, creating this rendered view:
+> Try to reproduce the markdown source code to create the following view:
```
Comment (sstevens2) [inline _episodes/markdown.md:206]: Image in the challenge has the python "hello everyone" code block on top from the example before the exercises in it as well as the other python/r/html code blocks.

File: _episodes/markdown.md [hunk 2]
```diff
@@ -266,10 +259,13 @@ You can define either
 > ~~~
 > {: .source }
 >
+<<<<<<< HEAD
+=======
 > to create the following rendered view, where both links point to <https://carpentries.org/>.
 >
 > ![Preview tab of inline and reference style links](../fig/exercise_links.png)
 >
+>>>>>>> gh-pages
```
Comment (sstevens2) [inline _episodes/markdown.md]: Looks like this still has the conflict in it...Think this suggestion should fix it?
```suggestion
> to create the following rendered view, where both links point to <https://carpentries.org/>.
>
> ![Preview tab of inline and reference style links](../fig/exercise_links.png)
>
```

**Review by sstevens2 — COMMENTED** [review]
Few edits suggested.  Looking good

File: _episodes/markdown.md (hunk 2 above)
Comment (annefou) [reply _episodes/markdown.md]: This one has been updated already!

File: _episodes/markdown.md (hunk 1 above)
Comment (annefou) [reply _episodes/markdown.md:206]: Well spotted!!! I jux fix it. Thanks

File: _episodes/markdown.md (hunk 2 above)
Comment (sstevens2) [reply _episodes/markdown.md]: must have been working on a previous commit

**Comment by sstevens2** [comment]
Looks good to me!  I'll go ahead and merge.

**Pull request merged**